### PR TITLE
✨ add `notEqual` and `NE` support.

### PR DESCRIPTION
**Pull request by Gaubee**
I'm developing the [ts-bigint-transform-jsbi](https://github.com/Gaubee/ts-bigint-transform-jsbi) repository for converting the **bigint** code of Typescript into **jsbi**.
For example, it is very simple to convert `ts. SyntaxKind. EqualsEqualsEqualsToken'into `JSBI. equal'.
So I also need `JSBI. noEqual'and `JSBI. NEQ' to handle `ExclamationEqualsEqualsToken'and `ExclamationEqualsToken'.`
The two API will make the **jsbi** more easy to use. so why no 😄 .

### CODE REVIEW

**Comment by jakobkummerow** [comment]
I'm fine with this. Let's call the new additions `notEqual` (add a 't') and `NE` (drop the 'Q').

Please also prepare a PR for https://github.com/GoogleChromeLabs/babel-plugin-transform-jsbi-to-bigint to transpile `JSBI.notEqual` and `JSBI.NE` to the corresponding native syntax `!==` and `!=`.

**Comment by Gaubee** [comment]
bingo GoogleChromeLabs/babel-plugin-transform-jsbi-to-bigint#4

**Comment by mathiasbynens** [comment]
Could you please also update the README? These new methods need to be documented.

**Comment by jakobkummerow** [comment]
Thanks, looks good now, will merge.

@mathiasbynens : any idea what's up with that build failure? Seems unrelated to this patch, I'd say?

**Comment by mathiasbynens** [comment]
I'll take a look later today.

**Comment by mathiasbynens** [comment]
@Gaubee Can you see `npm run lint` and commit any changes?

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "build": "rollup --config rollup.config.js",
     "dev": "rollup --config rollup.config.js --watch",
-    "test": "for file in tests/*.mjs; do node --no-warnings --experimental-modules \"${file}\"; done; set -e; for file in benchmarks/*.mjs; do node --no-warnings --experimental-modules \"${file}\"; done",
+    "test": "for file in tests/*.mjs; do node --no-warnings --experimental-modules ${file}; done; set -e; for file in benchmarks/*.mjs; do node --no-warnings --experimental-modules ${file}; done",
```
Comment (mathiasbynens) [inline package.json]: Why was this changed?
